pkg/machine/e2e/basic_test.go

```diff
@@ -93,6 +96,29 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(out).ToNot(ContainSubstring("gvproxy"))
 	})
 
+	It("podman volume on non-standard path", func() {
+		dir, err := os.MkdirTemp("", "machine-volume")
+		Expect(err).ToNot(HaveOccurred())
+		defer os.RemoveAll(dir)
+
+		testString := "abcdefg1234567"
+		testFile := "testfile"
+		err = os.WriteFile(filepath.Join(dir, testFile), []byte(testString), 0644)
+		Expect(err).ToNot(HaveOccurred())
+
+		name := randomString()
+		machinePath := "/does/not/exist"
+		init := new(initMachine).withVolume(fmt.Sprintf("%s:%s", dir, machinePath)).withImagePath(mb.imagePath).withNow()
+		session, err := mb.setName(name).setCmd(init).run()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(session).To(Exit(0))
+
+		bm := basicMachine{}
+		ls, err := mb.setCmd(bm.withPodmanCommand([]string{"run", "-t", "quay.io/libpod/alpine", "cat", filepath.Join(machinePath, testFile)})).run()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(ls).To(Exit(0))
+		Expect(ls.outputToString()).To(ContainSubstring(testString))
```

Comment:
How can this work? You are never mounting the volume in the container.
Also please do not use podman run here as this requires a image pull. Just use `podman machine ssh`  to check if the file is mounted at the correct location.